stl/inc/ranges

```diff
@@ -3068,6 +3068,14 @@ namespace ranges {
         requires view<_Vw> && input_range<range_reference_t<_Vw>>
     class join_view : public _Join_view_base<_Vw> {
         // clang-format on
+
+#ifndef _USE_JOIN_VIEW_INPUT_RANGE
+        static_assert(forward_range<_Vw>,
+            "Using join_view with input-only ranges is temporarily disabled because of a bug that makes join_view "
+            "misbehave for certain ranges. See https://cplusplus.github.io/LWG/issue3698. You can define "

```

Comment:
Saying "bug" might be confusing here. We're not referring to a bug in our implementation - instead we're referring to a (reported) bug in the Standard itself. Would rephrasing this to "because this can misbehave for certain ranges" be better?